src/kubernetes-ingress-controller/guides/using-tcpingress.md

```diff
@@ -175,7 +292,7 @@ Connected to 35.247.39.83.
 Escape character is '^]'.
 Welcome, you are connected to node gke-harry-k8s-dev-pool-1-e9ebab5e-c4gw.
 Running on Pod tcp-echo-844545646c-gvmkd.
-In namespace default.
+gtIn namespace default.
```

Comment:
why did here changed to `gtIn`?really from the response or just a typo?

Reply:
Vim hotkeys accidentally used in insert mode 🙂